src/data/queries/alertSingle.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  QueryData,
+  QueryFormatter,
+  QueryParams,
+  QueryOpts,
+} from 'next-drupal-query'
+import { drupalClient } from '@/lib/drupal/drupalClient'
+import { queries } from '.'
+import { ParagraphAlertSingle } from '@/types/drupal/paragraph'
+import { AlertSingle } from '@/types/formatted/alertSingle'
+
+export const params: QueryParams<null> = () => {
+  return queries
+    .getParams()
+    .addPageLimit(3)
```

Comment:
i don't think we want a page limit on this (just an example when gathering data from the `_playground` route)

Reply:
Good catch, fixed